KIPs/kip-81.md

```diff
@@ -1,9 +1,9 @@
 ---
 kip: 81
 title: Implementing the on-chain governance voting method
-author: Yeri<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+author: Yeri<yeriel.lee@klaytn.foundation>, Daniel<daniel.cc@klaytn.foundation>, Aidan<aidan.kwon@klaytn.foundation>, Ollie<ollie.j@klaytn.foundation>, Eddie<eddie.kim0@klaytn.foundation>
```

Comment:
Can you update this with github id instead of email?

Reply:
updated